src/Workspaces/Core/Portable/CodeRefactorings/FixAllOccurences/FixAllProviderInfo.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings
+{
+    /// <summary>
+    /// Contains computed information specific to FixAll for a given <see cref="CodeRefactoringProvider"/>, such as supported <see cref="FixAllScope"/>s.
+    /// </summary>
+    internal sealed class FixAllProviderInfo
```

Comment:
this class doesn't seem to have much value.  do we really need it?  whoever holds onto it could just hold onto these two fields?

also, consider a struct.

Reply:
This is held onto a map by provider in the CodeFixService/CodeRefactoringService: https://github.com/dotnet/roslyn/blob/e1629052738683dac234a4d75bb1d1c442c46d7d/src/Features/LanguageServer/Protocol/Features/CodeFixes/CodeFixService.cs#L55